schedule/yast/yast2_ncurses/yast2_ncurses_textmode@svirt-xen-hvm.yaml

```diff
@@ -9,5 +9,6 @@ schedule:
   - boot/boot_to_desktop
   - console/prepare_test_data
   - console/consoletest_setup
+  - console/yast2_bootloader
```

Comment:
let's order it in the same way, after `yast2_i` so whoever extends this in the future can notice that it might matter.

Reply:
done